extension-requests/local-utils.js

```diff
@@ -252,9 +252,15 @@ const addSpinner = (container) => {
   return removeSpinner;
 };
 
-const dateTimeString = (timestamp) => {
-  return new Date(timestamp).toISOString().substring(0, 16);
-};
+function dateTimeString(milliseconds) {
```

Comment:
- can we please add the return type of this function as a jsdoc comment the return type looks complex on the first look.